
Work in progress to get the work in progress xft support --without-mule aware
...

  - Rene Rebe <rene@exactcode.de>

--- xemacs/src/redisplay-x.c.orig	2004-12-10 07:43:50.000000000 +0100
+++ xemacs/src/redisplay-x.c	2005-01-06 13:15:12.145064072 +0100
@@ -172,9 +172,10 @@
 					  MULE is not defined */
   int runs_so_far = 0;
   int i;
+  int need_ccl_conversion = 0;
+
 #ifdef MULE
   struct ccl_program char_converter;
-  int need_ccl_conversion = 0;
 #endif
 
   for (i = 0; i < len; i++)
